src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue18452.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+[Category(UITestCategories.WebView)]
+public class Issue18452 : _IssuesUITest
+{
+	public override string Issue => "NullReferenceException throws on Windows when setting Cookies on .NET MAUI WebView";
+	string? expected = "Success";
+
+	public Issue18452(TestDevice device) : base(device)
+	{
+	}
+
+	[Test]
+	public void WebViewLoadedWithoutException()
```

Comment:
The test fails on Catalyst:
<img width="1187" alt="image" src="https://github.com/user-attachments/assets/5ace46cf-2703-4e60-8eac-fd136aa60bbb">